appinventor/components-ios/src/extensions/posenet.swift

```diff
@@ -0,0 +1,422 @@
+    return skeleton
+  }
+  
+  @objc open var Nose: [Double] {
+    return _keyPoints["nose"]!
+  }
+  
+  @objc open var LeftEye: [Double] {
+    return _keyPoints["leftEye"]!
+  }
+  
+  @objc open var RightEye: [Double] {
+    return _keyPoints["rightEye"]!
+  }
+  
+  @objc open var LeftEar: [Double] {
+    return _keyPoints["leftEar"]!
+  }
+  
+  @objc open var RightEar: [Double] {
+    return _keyPoints["rightEar"]!
+  }
+  
+  @objc open var LeftShoulder: [Double] {
+    return _keyPoints["leftShoulder"]!
+  }
+  
+  @objc open var RightShoulder: [Double] {
+    return _keyPoints["rightShoulder"]!
+  }
+  
+  @objc open var LeftElbow: [Double] {
+    return _keyPoints["leftElbow"]!
+  }
+  
+  @objc open var RightElbow: [Double] {
+    return _keyPoints["rightElbow"]!
+  }
+  
+  @objc open var LeftWrist: [Double] {
+    return _keyPoints["leftWrist"]!
+  }
+  
+  @objc open var RightWrist: [Double] {
+    return _keyPoints["rightWrist"]!
+  }
+  
+  @objc open var LeftHip: [Double] {
+    return _keyPoints["leftHip"]!
+  }
+  
+  @objc open var RightHip: [Double] {
+    return _keyPoints["rightHip"]!
+  }
+  
+  @objc open var LeftKnee: [Double] {
+    return _keyPoints["leftKnee"]!
+  }
+  
+  @objc open var RightKnee: [Double] {
+    return _keyPoints["rightKnee"]!
+  }
+  
+  @objc open var LeftAnkle: [Double] {
+    return _keyPoints["leftAnkle"]!
+  }
+  
+  @objc open var RightAnkle: [Double] {
+    return _keyPoints["rightAnkle"]!
+  }
+  
+  @objc open var BackgroundImage: String {
+      return _backgroundImage
+  }
+  
+  @objc open var Enabled: Bool {
+    get {
+      return _enabled
+    }
+    set {
+      _enabled = newValue
+      if _initialized {
+        do {
+          try assertWebView("Enabled")
+        } catch {
+          print(_ERROR_WEBVIEWER_NOT_SET, "Enabled")
+        }
+      }
+    }
+  }
+  
+  @objc open var UseCamera: String {
+    get {
+      return _cameraMode
+    }
+    set {
+      if newValue == _BACK_CAMERA || newValue == _FRONT_CAMERA {
+        _cameraMode = newValue
+        if _initialized {
+          do {
+            let frontFacing = (newValue == _FRONT_CAMERA)
+            try assertWebView("UseCamera", frontFacing)
+          } catch {
+            print(_ERROR_WEBVIEWER_NOT_SET, "UseCamera")
+          }
+        }
+      } else {
+        // No Error.Extension_Error?
+        _form?.dispatchErrorOccurredEvent(self, "UseCamera", 3300, "Invalid camera selection. Must be either 'Front' or 'Back'.")
+      }
+    }
+  }
+  
+  
+  // MARK: PoseNet Methods
+  
+  
+  @objc open func Initialize() {
+    if _webview != nil {
+      print("initialized")
+      _initialized = true
+    }
+  }
+  
+  @objc open func ModelReady() {
+    EventDispatcher.dispatchEvent(of: self, called: "ModelReady")
+    print("dispatched ModelReady")
+  }
+  
+  @objc open func Error(_ errorCode: AnyObject, _ errorMessage: AnyObject) { //TODO: What to do about anyobject? Should be int and String
```

Comment:
Make the types Int32 and String here, and then on the next line cast them `as AnyObject` because the dispatch function isn't interested in the types.